relayer-cli/src/state/11155111.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do we need this in Gihub? If so, move it outside the src/ folder to `relayer-cli/data`